catalog/ui/catalog-ui-search/pom.xml

```diff
@@ -351,7 +351,9 @@
                             !org.abego.treelayout.*,
                             *
                         </Import-Package>
-                        <Export-Package/>
+                        <Export-Package>
+                            org.codice.ddf.catalog.ui.metacard.workspace.transformer
```

Comment:
❗️As a general rule, we don't export classes, only interfaces, and put them in separate API bundles so we can evolve the API and implementation independently and not force clients to depend on an implementation. Curious to know if there's a reason why we're doing something different here.

Reply:
The classes important for export are the interfaces, mainly `WorkspaceTransformation`. I see your point, though. I'll factor it out into a new bundle.